tests/Smoke/InstallViaComposerTest.php

```diff
@@ -68,7 +69,16 @@ public function testInstallationIsPossible()
             json_encode($initialComposerFileState, Utils::calculateBitmask(array('JSON_PRETTY_PRINT')))
         );
 
-        $this->assertSame(0, CommandExecutor::create('composer install -q', $tmpPath)->getResult()->getCode());
+        try {
+            $this->assertSame(0, CommandExecutor::create('composer install -q', $tmpPath)->getResult()->getCode());
+        } catch (ProcessTimedOutException $e) {
```

Comment:
it doesn't cover situation when there is no `composer` available (eg under docker)

Reply:
that failure would not result in a `ProcessTimedOutException` I think?

Reply:
nope, that's why this `catch` doesn't cover it

Reply:
ack